test/dynamo/test_package.py

```diff
@@ -30,6 +30,50 @@
 def compute_loss_helper(x):
     return reduce_to_scalar_loss(x)
 
+class _tempTensorSamplerForQualName:
```

Comment:
Suggestion: Can we move this to inside the test definition since we don't use in other test cases? That way the class defn is closer to the test itself